tools/gencert/ca/genca.sh

```diff
@@ -10,7 +10,7 @@ openssl ecparam -genkey -name "$curve" -out "$keyname"
 # generate a "signing request"
 openssl req -new -key "$keyname" -out "$keyname".csr
 # self sign the request
-openssl x509 -req -in "$keyname".csr -signkey "$keyname" -out "$certname"
+openssl x509 -req -days 1825  -in "$keyname".csr -signkey "$keyname" -out "$certname"
```

Comment:
Your CA root should also be long-lived. [Yubico's U2F CA](https://developers.yubico.com/U2F/yubico-u2f-ca-certs.txt) expires in 2050, just like their attestation certificates do.

Reply:
Good point, I will update.